udp-reassembly-offload.md

```diff
@@ -38,11 +39,13 @@ URO can only be attempted on a batch of packets that meet **all** the following
 
 - 5-tuple matches.
 - Payload length is identical for all datagrams, except the last datagram which may be less.
-- The checksums on pre-coalesced packets must be correct.
-- ECN, DF bits must match on all packets (IPv4).
-- NextHeader must be UDP (IPv6).
+- The UDP checksums on pre-coalesced packets must be correct.
+- TTL, ToS/ECN, Protocol, and DF bit must match on all packets (IPv4).
+- TC/ECN, FlowLabel, and HopLimit must match, and NextHeader must be UDP (IPv6).
 - The total length of the Single Coalesced Unit (SCU) must not exceed IP max length.
 
+The coalesced IP length field and the UDP length field must be updated with the coalesced length. The coalesced IPv4 checksum field must be updated with the new length. The coalesced UDP checksum field is ignored and does not need to be updated (since it was already validated individually).
```

Comment:
The "IPv4 checksum field" has the new length?

Reply:
I mean the IPv4 checksum covers the updated length field.